CoinbasePro websocket crashes

Turning on the CoinbasePro websocket crashes the program (using default config.json):

[INFO]:  2019/04/03 22:39:12 CoinbasePro BTC-GBP SPOT: TICKER: Last $5265.32 USD (£3998.98 GBP) Ask $0.00 USD (£0.00 GBP) Bid $0.00 USD (£0.00 GBP) High $5338.61 USD (£4054.65 GBP) Low $4793.34 USD (£3640.52 GBP) Volume 1928.81197730
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x875be7]

goroutine 621 [running]:
github.com/thrasher-/gocryptotrader/exchanges/coinbasepro.(*CoinbasePro).ProcessSnapshot(0xc0000bfd40, 0xc000660690, 0x3fe00, 0xa9fb60)
        /home/jeee/gocryptotrader/exchanges/coinbasepro/coinbasepro_websocket.go:214 +0x157
github.com/thrasher-/gocryptotrader/exchanges/coinbasepro.(*CoinbasePro).WsHandleData(0xc0000bfd40)
        /home/jeee/gocryptotrader/exchanges/coinbasepro/coinbasepro_websocket.go:176 +0x69b
created by github.com/thrasher-/gocryptotrader/exchanges/coinbasepro.(*CoinbasePro).WsConnect
        /home/jeee/gocryptotrader/exchanges/coinbasepro/coinbasepro_websocket.go:87 +0x313

--- a/exchanges/coinbasepro/coinbasepro_websocket.go
+++ b/exchanges/coinbasepro/coinbasepro_websocket.go
@@ -199,7 +199,7 @@ func (c *CoinbasePro) WsHandleData() {
 
 // ProcessSnapshot processes the initial orderbook snap shot
 func (c *CoinbasePro) ProcessSnapshot(snapshot *WebsocketOrderbookSnapshot) error {
-	var base *orderbook.Base
+	var base orderbook.Base
 	for _, bid := range snapshot.Bids {
 		price, err := strconv.ParseFloat(bid[0].(string), 64)
 		if err != nil {
@@ -234,7 +234,7 @@ func (c *CoinbasePro) ProcessSnapshot(snapshot *WebsocketOrderbookSnapshot) erro
 	base.AssetType = "SPOT"
 	base.Pair = pair
 
-	err := c.Websocket.Orderbook.LoadSnapshot(base, c.GetName(), false)
+	err := c.Websocket.Orderbook.LoadSnapshot(&base, c.GetName(), false)
 	if err != nil {
 		return err
 	}

